frontend/src/types/AssessmentTypes.ts

```diff
@@ -1,10 +1,29 @@
+import { Grade } from "../APIClients/types/UserClientTypes";
+
 export type StatusProperty = "DRAFT" | "PUBLISHED" | "ARCHIVED" | "DELETED";
 
+export enum TestTypes {
+  Beginning = "BEGINNING",
+  End = "END",
+}
```

Comment:
```suggestion
export enum AssessmentTypeEnum {
  BEGINNING = "BEGINNING",
  END = "END",
}
```
sorry for making you rename again lol, but to be consistent with backend

Reply:
Looks like it was changed in another PR so I'll leave to its new name